src/components/DailyTasks.js

```diff
@@ -1,48 +1,84 @@
-import {useEffect, useState} from 'react';
+import {useState, useEffect} from 'react';
+import {dailyTodos} from '../db';
 import {loadFromLocalStorage, writeToLocalStorage} from '../util/localstorage';
-import styled from 'styled-components';
 import TaskList from './TaskList';
-import dailyTasks from '../db';
 
 export default function DailyTasks() {
+  const [daily, monday, tuesday, wednesday, thursday, friday, weekend] = dailyTodos;
+  let currentday;
+  switch (new Date().getDay()) {
+    case 0:
+      currentday = [weekend];
+      break;
+    case 1:
+      currentday = [daily, monday];
+      break;
+    case 2:
+      currentday = [daily, tuesday];
+      break;
+    case 3:
+      currentday = [daily, wednesday];
+      break;
+    case 4:
+      currentday = [daily, thursday];
+      break;
+    case 5:
+      currentday = [daily, friday];
+      break;
+    case 6:
+      currentday = [weekend];
+      break;
+    default:
+  }
```

Comment:
To simplify this, I would recommend to add another property to your data, maybe something like `weekday` which holds a value from 1 to 7. You could then go through your `dailyTodos` array and find the object with the `weekday` which corresponds to the actual date. Something like this:
```js
const weekdayToday = new Date().getDay();
const todaysTodos = dailyTodos.find(todo => todo.weekday === weekdayToday + 1);
```  
I haven't tested it, but I think it should work like this and this saves you a few lines of code. Also this way is a bit safer since you wouldn't rely on array destructuring to get the correct data.

Reply:
find() does not work here. I was able to implement it with filter(), thanks! 
From the function `new Date().getDay();` I get a number from 0 to 6, I do not understand why I should not continue to work with it? Thanks for your feedback :)

Reply:
It might be easier to work with the days as we know and count them - starting the week at monday (1). But it is entirely up to you :)